index.html

```diff
@@ -246,22 +246,36 @@ <h1>contact me</h1>
           that you need built or a project that needs coding. I'd love to help
           with it
         </p>
-        <p>
-          <input type="text" id="name" value="Christian Biringanine" />
-        </p>
-        <p>
-          <input type="email" id="email" value="christianbiringanine22@gmail" />
-        </p>
-        <p>
+        <form
+          action="https://formspree.io/f/mzbovqko"
+          method="POST"
+          class="form-contact"
+        >
+          <input
+            type="text"
+            id="name"
+            name="name"
+            placeholder="Christian Biringanine"
+            maxlength="30"
+            required
+          /><br />
+          <input
+            type="email"
+            id="email"
+            name="mail"
+            placeholder="christianbiringanine22@gmail"
+            required
+          /><br />
           <textarea
```

Comment:
- [ ] please don't use your own name as placeholder text, what is intended with placeholder text is for example "full name" in the name input and "email address" in the mail input field.

Reply:
Alright, @mikethreels I got you. let me change that real quick.